docs/Reference.md

```diff
@@ -1268,6 +1269,25 @@ Example:
 
 ---
 
+### `limitsNavigationsToAppBoundDomains`[⬆](#props-index)<!-- Link generated with jump2header -->
+
+If true indicates to WebKit that a WKWebView will only navigate to app-bound domains. Only applicable for iOS 14 or greater.
+
+Once set, any attempt to navigate away from an app-bound domain will fail with the error “App-bound domain failure.”
```

Comment:
"If true" implies it, but it would be nice to explicitly specify that the default value is false.

Also, does the "App-bound domain failure" error message come through react-native-webview's `onError` event?

Reply:
Sorry, I seem to have forgot to answer here.

> "If true" implies it, but it would be nice to explicitly specify that the default value is false.

It is a new value that's being added. It's barely documented. At this point I would not be surprised if setting it to false behaves differently from leaving out the attribute. For this reason alone, I am not comfortable adding a default value.

> Also, does the "App-bound domain failure" error message come through react-native-webview's onError event?

I don't remember at this point and can't double check right now. I think it did, yes.